index.bs

```diff
@@ -175,12 +175,12 @@ state that is considered not compatible with Picture-in-Picture.
 When the <dfn>exit Picture-in-Picture algorithm</dfn> is invoked,
 the user agent MUST run the following steps:
 
-1. Let |pipWindow| be the current Picture-in-Picture window.
-2. Let |pipVideo| be the video contained in the |pipWindow|.
-3. Let |video| be the source video of |pipVideo| frames.
-4. Close |pipWindow|.
-5. If |pipWindow| can't be closed, throw a {{UnknownError}} and abort these
-    steps.
+1. If {{pictureInPictureElement}} is null, throw a {{InvalidStateError}} and abort
+    these steps.
+2. Let |pipWindow| be the current Picture-in-Picture window.
+3. Let |pipVideo| be the video contained in the |pipWindow|.
+4. Let |video| be the source video of |pipVideo| frames.
```

Comment:
This difference between |pipVideo|, |video| and |pictureInPictureElement| is weak. Probably orthogonal to this change though.